src/index.js

```diff
@@ -1,8 +1,9 @@
+/* eslint-disable */
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+
+import App from './app';
```

Comment:
- [ ] There's a module not found error because the file `app.js` cannot be found. Please use `import App from ./App` instead 👍
![image](https://user-images.githubusercontent.com/46839548/197811979-4e22deba-fc82-44a0-bd9a-ea299aa4cf1f.png)

Reply:
Thanks for pointing that out. 😄